src/pyramid/view.py

```diff
@@ -216,6 +217,12 @@ def __init__(self, **settings):
             if settings.get('context') is None:
                 settings['context'] = settings['for_']
         self.__dict__.update(settings)
+        self._get_info()
+
+    def _get_info(self):
+        frameinfo = inspect.stack()[2]
+        sourceline = frameinfo[4][0].strip()
+        self._info = frameinfo[1], frameinfo[2], frameinfo[3], sourceline
```

Comment:
should this use the `_depth` argument at all? The `[2]` in the stack may not be what you want in all situations but I'm not sure? I suspect this will always point at the location of the view_config decorator, but if view_config is wrapped in something else, you'd want to point there?

Reply:
Ah,. that's a good point. It should use depth + 1.